packages/exception/src/http/http.exception.ts

```diff
@@ -2,11 +2,23 @@ import { ErrorMessage } from '@exception/error-message.interface';
 import { ErrorPayload } from '@exception/error-payload.interface';
 import { InternalException } from '@exception/internal.exception';
 
+export interface SerializedHTTPException {
+  message: string;
+  path: string;
+  statusCode: number;
+  statusText: string;
+  timestamp: string;
+}
+
 export class HTTPException extends InternalException {
```

Comment:
shouldnt we make HTTPException implement SerializedHTTPException?

Reply:
Yes